CMakeLists.txt

```
@@ -19,6 +19,7 @@ set(SNITCH_MAX_PATH_LENGTH          1024 CACHE STRING "Maximum length of a file
 # Feature toggles.
 set(SNITCH_DEFINE_MAIN                     ON  CACHE BOOL "Define main() in snitch -- disable to provide your own main() function.")
 set(SNITCH_WITH_EXCEPTIONS                 ON  CACHE BOOL "Use exceptions in snitch implementation -- will be forced OFF if exceptions are not available.")
+set(SNITCH_WITH_MULTITHREADING             ON  CACHE BOOL "Multi-threading support and thread local variables -- disable if needed.")
```

I would recommend rewording the help text for clarity. Similar change for the meson scripts. 
```suggestion
set(SNITCH_WITH_MULTITHREADING             ON  CACHE BOOL "Make the testing framework thread-safe -- disable if multithreading is not needed.")
```